<<<<<<< HEAD
From b532add2d1287489d4541e79c976bb77795696cd Mon Sep 17 00:00:00 2001
From: Ember Keske <git@n0emis.eu>
Date: Wed, 2 Aug 2023 06:36:46 +0200
=======
From c46323dfc4a3d5bf17f7508f5b5f223858a598c3 Mon Sep 17 00:00:00 2001
From: Maximilian Bosch <maximilian@mbosch.me>
Date: Tue, 22 Dec 2020 15:39:15 +0100
>>>>>>> 903308adb4b (Improved error handling, differentiate nix/non-nix networks)
Subject: [PATCH 2/2] Remove check for config-file as it's stored in /etc/grocy

---
 helpers/PrerequisiteChecker.php | 1 -
 1 file changed, 1 deletion(-)

diff --git a/helpers/PrerequisiteChecker.php b/helpers/PrerequisiteChecker.php
<<<<<<< HEAD
index da431b4b..6b878627 100644
--- a/helpers/PrerequisiteChecker.php
+++ b/helpers/PrerequisiteChecker.php
@@ -17,7 +17,6 @@ class PrerequisiteChecker
 	public function checkRequirements()
 	{
 		self::checkForPhpVersion();
=======
index cbe0a3a..a484db2 100644
--- a/helpers/PrerequisiteChecker.php
+++ b/helpers/PrerequisiteChecker.php
@@ -11,7 +11,6 @@ class PrerequisiteChecker
 {
 	public function checkRequirements()
 	{
>>>>>>> 903308adb4b (Improved error handling, differentiate nix/non-nix networks)
-		self::checkForConfigFile();
 		self::checkForConfigDistFile();
 		self::checkForComposer();
 		self::checkForPhpExtensions();
-- 
<<<<<<< HEAD
2.41.0
=======
2.29.2
>>>>>>> 903308adb4b (Improved error handling, differentiate nix/non-nix networks)

